docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`2442` Users will now see the accounts in the assets page sorted by name.
```

Comment:
That's not correct though. It keeps them sorted by amount by default, right? Which is good! It's the default behaviour I want.

If I understand the bug that's fixed then shouldn't the changelog read:

"When users sort by account in the assets page, sorting will run on the account name and not the account address"

Right?

Reply:
It had more words when I was thinking it, to be honest, it seems that some of them didn't make it to the changelog :P

Reply:
yeah i can imagine. Just tell me when you change it to approve.

Reply:
Changed